content/docs/start/data-management/metrics-parameters-plots.md

```diff
@@ -43,8 +49,7 @@ $ dvc repro
 ```
 
 [earlier pipeline]: /doc/start/data-management/data-pipelines
-
-<details>
```

Comment:
@omesser Did you mean to drop the `<details>` here? If so, I think you need to clean up some of the section below.